### ISSUE TITLE
Title Style Updates

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1350]
**Component Block:** None (styles applied via heading sizes in word)

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=132-797&t=7ZXrRUQI9M74s5dW-0

As a Web Liaison, I want the ability to select a heading size for a new section of the web page, so that I can follow SEO organization and create a natural break in page content. 

**Requirements:** 
All heading titles should be updated globally to follow the brand guidelines styling: 
H1 PP Neue Montreal, 48px, bold
H2 PP Neue Montreal, 36px, bold
H3 PP Neue Montreal, 24px, bold
H4 PP Neue Montreal, 18px, bold
H5 PP Neue Montreal, 16px, bold
H6 PP Neue Montreal, 14px, bold
All headings should have appropriate aria labels/tags for a screen reader to announce new heading/section
If heading configured as a link, then link is keyboard navigable and screen reader appropriately announces the link (**best practice is to not have titles as links, and we will move away from this practice in content strategy in Phase 4)
Title colors: black and primary color

Mobile sizes heading titles down and should match the following: 
H1 PP Neue Montreal, 36px, bold
H2 PP Neue Montreal, 24px, bold
H3 PP Neue Montreal, 18px, bold
H4 PP Neue Montreal, 16px, bold
H5 PP Neue Montreal, 14px, bold
H6 PP Neue Montreal, 14px, bold

**Acceptance Criteria:** 
1. Given I am applying an H1 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 48px font size, and bold styling to match brand guidelines
2. Given I am applying an H2 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 36px font size, and bold styling to match brand guidelines
3. Given I am applying an H3 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 24px font size, and bold styling to match brand guidelines
4. Given I am applying an H4 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 18px font size, and bold styling to match brand guidelines
5. Given I am applying an H5 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 16px font size, and bold styling to match brand guidelines
6. Given I am applying an H6 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 14px font size, and bold styling to match brand guidelines
7. Given I am accessing a web page with a screen reader, when the screen reader announces the page content, then it will announce a new heading/section of the page
8. Given I am accessing a page with a title configured as a hyperlink, when I am navigating via my keyboard, then I am able to access and click on the hyperlinked title. 
9. Given I am accessing a page with a title configured as a hyperlink, when I am using a screen reader, then the screen reader properly announces the title as a hyperlink.
10. Given I am authoring a title component, when I want to create contrast on the page, then I am able to change the title color to primary.
11. Given I am viewing an H1 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 36px, bold. 
12. Given I am viewing an H2 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 24px, bold. 
13. Given I am viewing an H3 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 18px, bold.
14. Given I am viewing an H4 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 16px, bold.  
15. Given I am viewing an H5 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 14px, bold. 
16. Given I am viewing an H6 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 14px, bold.

**Comment by aaronstewart-dev** [comment]
@beccasuchanek Pushed title fixes to develop.

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

Test Page URL: https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/title-style

1. ✅ Given I am applying an H1 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 48px font size, and bold styling to match brand guidelines **See testing above in comment**
2. ✅ Given I am applying an H2 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 36px font size, and bold styling to match brand guidelines **See testing above in comment**
3. ✅ Given I am applying an H3 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 24px font size, and bold styling to match brand guidelines **See testing above in comment**
4. ✅ Given I am applying an H4 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 18px font size, and bold styling to match brand guidelines **See testing above in comment**
5. ✅ Given I am applying an H5 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 16px font size, and bold styling to match brand guidelines **See testing above in comment**
6. ✅ Given I am applying an H6 style to a heading title, when the content loads, then the heading is styled as PP Neue Montreal font, at 14px font size, and bold styling to match brand guidelines **See testing above in comment**
7. ✅ Given I am accessing a web page with a screen reader, when the screen reader announces the page content, then it will announce a new heading/section of the page **See testing above in comment**
8. ✅ Given I am accessing a page with a title configured as a hyperlink, when I am navigating via my keyboard, then I am able to access and click on the hyperlinked title. **See testing above in comment**
9. ✅ Given I am accessing a page with a title configured as a hyperlink, when I am using a screen reader, then the screen reader properly announces the title as a hyperlink. **See testing above in comment**
10. ✅ Given I am authoring a title component, when I want to create contrast on the page, then I am able to change the title color to primary. 

<img width="480" height="785" alt="Image" src="https://github.com/user-attachments/assets/1a059d25-144a-47bb-a49c-2ab946572473" />

11. ✅ Given I am viewing an H1 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 36px, bold. **See testing above in comment**
12. ✅ Given I am viewing an H2 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 24px, bold. **See testing above in comment**
13. ✅ Given I am viewing an H3 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 18px, bold. **See testing above in comment**
14. ✅ Given I am viewing an H4 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 16px, bold. 

<img width="1695" height="831" alt="Image" src="https://github.com/user-attachments/assets/c6394be6-a928-45a6-aa71-70eeb3fca3da" />

15. ✅ Given I am viewing an H5 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 14px, bold.

<img width="1755" height="894" alt="Image" src="https://github.com/user-attachments/assets/a10199c7-f2b6-4ce1-acbd-3d0882d85741" />

16. ✅ Given I am viewing an H6 on mobile, when the block renders, then the heading is styled as PP Neue Montreal, 14px, bold.

<img width="1745" height="865" alt="Image" src="https://github.com/user-attachments/assets/bbc72a51-d533-4a96-8481-be41fc367ded" />
